docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -44,81 +39,186 @@ Specifically, this guide will have you do the following:
+```
+wget https://raw.githubusercontent.com/projectcalico/calico-cni/k8s-1.1-docs/samples/kubernetes/master/dns/skydns.yaml
+kubectl create -f skydns.yaml
+```
 
-First, boot up the node machine using the bootable ISO we downloaded earlier.  You should be automatically logged in as the `core` user.
+## Install the Kubernetes UI Addon (Optional)
 
-Make a copy of the `node-config-template.yaml` in the `calico-kubernetes` repository for this machine.
+The Kubernetes UI can be installed using `kubectl` to run the following manifest file.
 
 ```
-cp calico-kubernetes-master/config/cloud-config/node-config-template.yaml node-config.yaml
+wget https://raw.githubusercontent.com/projectcalico/calico-cni/k8s-1.1-docs/samples/kubernetes/master/kube-ui/kube-ui.yaml
+kubectl create -f kube-ui.yaml
 ```
 
-You'll need to replace the following variables in the `node-config.yaml` file to match your deployment.
-- `<HOSTNAME>`: Hostname for this node (e.g. kube-node1, kube-node2)
-- `<SSH_PUBLIC_KEY>`: The public key you will use for SSH access to this server.
-- `<KUBERNETES_MASTER>`: The IPv4 address of the Kubernetes master.
+## Launch other Services With Calico-Kubernetes
+
+At this point, you have a fully functioning cluster running on Kubernetes with a master and two nodes networked with Calico. You can now follow any of the [standard documentation](../../../examples/) to set up other services on your cluster.
+
+## Connectivity to outside the cluster
 
-Move the modified `node-config.yaml` to your Kubernetes node machine and install and configure CoreOS on the node using the following command.
+Because containers in this guide have private `192.168.0.0/16` IPs, you will need NAT to allow connectivity between containers and the internet. However, in a production data center deployment, NAT is not always necessary, since Calico can peer with the data center's border routers over BGP.
+
+### NAT on the nodes
+
+The simplest method for enabling connectivity from containers to the internet is to use outgoing NAT on your Kubernetes nodes.
+
+Calico can provide outgoing NAT for containers.  To enable it, use the following `calicoctl` command:
 
 ```
-sudo coreos-install -d /dev/sda -C stable -c node-config.yaml
+ETCD_AUTHORITY=<master_ip:6666> calicoctl pool add <CONTAINER_SUBNET> --nat-outgoing
 ```
 
-Once complete, eject the bootable disc and restart the server.  When it comes back up, you should have SSH access as the `core` user using the public key provided in the `node-config.yaml` file.  It will take some time for the node to be fully configured.  Once fully configured, you can check that the node is running with the following command on the Kubernetes master.
+By default, `<CONTAINER_SUBNET>` will be `192.168.0.0/16`.  You can find out which pools have been configured with the following command:
 
 ```
-kubectl get nodes
+ETCD_AUTHORITY=<master_ip:6666> calicoctl pool show
 ```
 
+### NAT at the border router
```

Comment:
Maybe this - https://github.com/projectcalico/calico-containers/blob/master/docs/bgp.md

Or some of the more general stuff on read the docs